en/formulaic_report_violation.md

```diff
@@ -0,0 +1,54 @@
+# Report a Violation
```

Comment:
Shouldn't we include Formulaic in the title of the page, e.g. `Formulaic — Report a violation` or similar?

Reply:
We could say "Report a Violation on Formulaic".  For the time being, the plan is that this page would only be accessible from Formulaic, so it should be clear which product this relates to.  But I don't see any reason not to make it clearer.